harness/determined/cli/task.py

```diff
@@ -217,6 +301,105 @@ def logs(args: Namespace) -> None:
+                        "-i",
+                        "--include",
+                        action="append",
+                        default=[],
+                        type=Path,
+                        help=ntsc.INCLUDE_DESC,
+                    ),
+                    Arg("--project_id", type=int, help="place this task inside this project"),
+                    Arg("--config", action="append", default=[], help=ntsc.CONFIG_DESC),
+                    Arg(
+                        "-f",
+                        "--follow",
+                        action="store_true",
+                        help="follow the logs of the task that is created",
+                    ),
+                    Arg("--fork", type=str, help="id of parent task to fork from"),
+                    Arg(
+                        "-p",
+                        "--parent",
+                        type=str,
+                        help="task id of parent task",
+                    ),
+                    Arg(
+                        "--inherit_context",
+                        action="store_true",
+                        help="inherit the context directory of the parent task (--parent flag required)",
+                    ),
+                ],
+            ),
+            Cmd(
+                "config",
+                config,
+                "get config for given task",
+                [
+                    Arg("task_id", type=str, help="ID of task to pull config from"),
+                    Arg(
+                        "--json",
+                        action="store_true",
+                        help="return config in JSON format",
+                    ),
+                ],
+            ),
+            Cmd(
+                "fork",
+                fork,
+                "fork task",
+                [
+                    Arg("parent_task_id", type=str, help="Id of parent task to fork from"),
+                    Arg(
+                        "-f",
+                        "--follow",
+                        action="store_true",
+                        help="follow the logs of the task that is created",
+                    ),
+                    Arg("--project_id", type=int, help="place this task inside this project"),
+                ],
+            ),
+            Cmd(
+                "kill",
+                kill,
+                "kill task",
+                [
+                    Arg("task_id", type=str, help=""),
+                    Arg(
+                        "--root",
+                        action="store_true",
+                        help="",
+                    )
+                ],
+            ),
+            Cmd(
+                "pause",
+                pause,
+                "pause task",
+                [
+                    Arg("task_id", type=str, help=""),
+                ],
+            ),
+            Cmd(
+                "resume unpause",
+                resume,
+                "resume or unpause a task",
```

Comment:
I understand that a long time ago we did `pause/activate` as the contrasting action pair, and we recently added `unpause` as an alias for `activate`.  But this feels like a third direction, with `resume` as the opposite of `pause`.  So the end user can pick from `unpause`, `resume`, or `activate` to achieve their goal but they have to pick differently based on different contexts.

That's really unfortunate, and we should pick one naming convention and stick to it.

Reply:
which ones would you like to see? for me, "resume" and "unpause" make the most sense, while "activate" is far from obvious.

Reply:
I agree.  I just want us to have a semblance of cohesiveness in our CLI naming.

I'm fine with either of of:
- deprecate `activate` and `unpause` for experiments and add `resume` to experiments, and add only `resume` here.
- leave experiments alone and add only `unpause` here

But I really don't like having aliases in the CLI other than as a temporary migration step.  We should have one official name for every action, and the webui and cli and docs should be aligned on what that name is, and multiple CLI subcommands should be aligned on what that name is.  I'm fine with migration periods where that isn't the case yet, but we should at least have to be moving in that direction, I think.

Since this is a new endpoint, I think there is no reason at all why one endpoint should have two names.

Reply:
>leave experiments alone and add only unpause here

I'd support this path, i.e. remove `resume` alias here.

Reply:
Updated API and command name